container.te

```diff
@@ -194,6 +194,21 @@ allow container_runtime_domain self:netlink_kobject_uevent_socket create_socket_
 allow container_runtime_domain self:netlink_tcpdiag_socket create_netlink_socket_perms;
 allow container_runtime_domain self:netlink_socket create_socket_perms;
 
+# Fix issue #282 (https://github.com/containers/container-selinux/issues/282)
+optional_policy(`
+	gen_require(`
+		type init_t;
+		type shadow_t;
+		type systemd_logind_t;
+		class process siginh;
+		class capability net_admin;
+	')
+
+	allow init_t container_user_t:process siginh;
+	allow init_t shadow_t:file { open read };
```

Comment:
Don't add this or the next line stick to container_user_t. and see if that works.

Reply:
It doesn't work but AVC shows nothing even though I ran `sudo semodule -DB`

Reply:
If setenforce 0, and it still fails, then it is not SELinux blocking access.

What podman command are you running?